core-bundle/src/Command/UserListCommand.php

```diff
@@ -86,10 +86,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return 0;
     }
 
-    /**
-     * @return Collection|array<UserModel>|UserModel|null
-     */
-    private function getUsers(bool $onlyAdmins = false)
+    private function getUsers(bool $onlyAdmins = false): Collection
```

Comment:
`Model::findBy()` can also return a single model: https://github.com/contao/contao/blob/555f24aaff4a5ebf330f61bda4c97e2cff6dee70/core-bundle/src/Resources/contao/library/Contao/Model.php#L954

Reply:
But not in this case, and the code that uses this method requires it to return a `Collection`.